26-register_registration/creator_navigates_to_description_tab.feature

```diff
@@ -11,6 +11,7 @@ Feature: Creator navigates to Description tab
       | Description                  |
       | Date published               |
       | Keywords                     |
+      | Vocabularies                 |
```

Comment:
det er brukt `Vocabulary settings`, `settings for Vocabularies` og `Vocabulary`. Mulig å bruke samme begrep?

Reply:
foreslår `Vocabulary settings` for admin, og `Vocabulary` som feltnavn

Reply:
kan være flere da, så tenker `Vocabulary settings` og `Vocabulary`